tests/unit/array/mixins/test_find.py

```diff
@@ -20,7 +20,7 @@
     'query',
     [np.random.random(32), np.random.random((1, 32)), np.random.random((2, 32))],
 )
-def test_find(storage, config, limit, query, start_storage):
+def test_find(storage, config, limit, query):
```

Comment:
I think start_storage should be added back

Reply:
definetly